File: src/app/servicios/modelos/servicio.ts
export interface Servicio {
  id: number;
  nombre: string;
  descripcion: string;
  precio: number;
  imagen: string | null;
  fechaCreacion: string;
  disponible: boolean;
  categoria: {
    id: number;
    nombre: string;
  };
}

export interface ServicioForm {
  nombre: string;
  descripcion: string;
  precio: number;
  disponible: boolean;
  categoriaId: number;
}

File: PeluqueriaReservacionesSolucion/peluqueria.reservaciones.Core/Dominio/Servicio.cs

namespace peluqueria.reservaciones.Core.Dominio
{
    public class Servicio
    {
        public int Id { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public int DuracionMinutos { get; set; }
        public decimal Precio { get; set; }
        public int CategoriaId { get; set; } 
        public bool Disponible { get; set; }
    }
}